index.html

```diff
@@ -0,0 +1,220 @@
+                    <li><a href="#">English</a></li>
+                    <li><a href="#" class='login-btn'>Mypage</a></li>
+                    <li><a href="#" class='login-btn'>Logout</a></li>
+                </ul>
+            </div>
+        </div>
+        <!-- End Top Nav -->
+
+        <!-- Navbar -->
+        <nav id='navbar' class="navbar">
+            <div class="container navbar-container">
+                <div class="navbar-header">
+                    <a href="index.html" class="navbar-brand logo">GLOBAL<span>SUMMIT</span></a>
+                </div>
+                <div class="navbar-right">
+                    <ul class="navbar-nav">
+                        <li class='nav-item'><a href="index.html">Home</a></li>
+                        <li class='nav-item'><a href="#program">Program</a></li>
+                        <li class='nav-item'><a href="#feature">Feature</a></li>
+                        <li class='nav-item'><a href="#partner">Sponsor</a></li>
+                        <li class='nav-item'><a href="about.html">About</a></li>
+                        <li class='nav-item'><a href="#" class="nav-btn">Ew campaign</a></li>
+                    </ul>
+                </div>
+                <div class="menu-btn">
+                    <div class="menu-btn__burger"></div>
+                </div>
+            </div>
+        </nav>
+    </div>
+    <!-- Main content -->
+    <div class="main">
+        <!-- hero section -->
+        <section class="hero">
+            <div class="overlay">
+                <div class="hero-bg">
+                    <div class="container">
+                        <div class="hero-content">
+                            <h3 class="hero-welcome-text">"Hello! Sharing world"</h3>
+                            <h1>THE VALUE OF <br> COLLABORATION IN<br>A REMOTE ENVIRONMENT.</h1>
+                            <p class="hero-p">
+                                A fun festival where people from more than 80 countries
+                                who believe in the value of sharing and openness and create
+                                positive change gather and share ideas will be held in Montreal in September.
+                            </p>
+                            <h2 class="hero-time">2022.09.21 (Wed) ~ 28 (Sat)</h2>
+                            <span class="hero-venue">@ National Museum of Canada, Art Center Montreal, etc.</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- program section -->
+        <section id="program" class="program">
+            <div class="program-bg">
+                <div class="container">
+                    <div class="program-content">
+                        <h2 class="section-title">Main Program</h2>
+                        <div class="program-list">
+                            <div class="program-item">
```

Comment:
- [ ] Kindly change the `div` tags with the class of `program-item` to `article` tags as requested by the previous reviewer.